pinc/Project.inc

```diff
@@ -901,6 +901,7 @@ class Project
             $this->can_be_managed_by_current_user
             || $this->PPVer_is_current_user
             || ($this->PPer_is_current_user && user_has_DU_access())
+            || User::current_username() == 'pg-automation'
```

Comment:
Is this necessary? The plan is for this username to be used only to give PG scripts the ability to see the clearance when accessed via API. This user should not be permitted to actually manage a project.

Reply:
Yes, this controls if the current user can see the clearance line and used by the API, and the project page, to restrict the clearance from the output if the user can't. This isn't used to control managing a project (that's `Project-> can_be_managed_by_user()` further up).